.gitignore

```diff
@@ -56,3 +56,16 @@ aws/lambda/log-classifier/data/dataset/
 
 # We use yarn.lock instead of package-lock.json for torchci
 /torchci/package-lock.json
+
+# Rust stuff
+# Generated by Cargo
+# will have compiled files and executables
+debug/
+target/
```

Comment:
any way to make these ignores a little bit more targeted (e.g. to specific folders)